docs/platform-services/automation-service/app-central/integrations/index.md

```diff
@@ -1,9 +1,8 @@
 ---
 slug: /platform-services/automation-service/app-central/integrations
-title: Available Integrations
-sidebar_label: Available Integrations
+title: Integrations in App Central
 sidebar_position: 1
-description: This article lists all the integrations available in App Central.
+description: ''
```

Comment:
was this intentional?

Reply:
Yes. @salvatoremosca wanted there to be no description on it so that the card in the list on the index page was the same as the others on the page. As he wrote in a DM to me:
> Do you know if we can remove the description under the Available Integrations tab so that the tabs are all the same size?